manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/cache.yaml

```diff
@@ -0,0 +1,235 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cache-deployer-statefulset
+  labels:
+    app: cache-deployer
+    app.kubernetes.io/name: {{ .Release.Name }}
+spec:
+  replicas: 1
+  serviceName: cache-deployer
+  selector:
+    matchLabels:
+      app: cache-deployer
+      app.kubernetes.io/name: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: cache-deployer
+        app.kubernetes.io/name: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: main
+        image: {{ .Values.images.cachedeployer }}
+        imagePullPolicy: Always
```

Comment:
"always" can work, actually I prefer it.
Would prefer follow other existing imagePullPolicy for uniform behavior. remember others are not. Would you double check?

Reply:
Other deployments are using imagePullPolicy: IfNotPresent. But for statefulset "Always" is the only support value.